src/Libraries/RevitNodes/Elements/Wall.cs

```diff
@@ -209,6 +209,22 @@ public static Wall ByCurveAndLevels(Autodesk.DesignScript.Geometry.Curve c, Leve
             return ByCurveAndHeight(c, height, startLevel, wallType);
         }
 
+        /// <summary>
+        /// Creates a Wall following the geometry of a surface.
+        /// </summary>
+        /// <param name="location"></param>
+        /// <param name="wallType"></param>
+        /// <param name="surface"></param>
+        /// <returns></returns>
+        public static FaceWall ByFace(string location, WallType wallType, Autodesk.DesignScript.Geometry.Surface surface)
```

Comment:
Actually the returned FaceWall won't be of much use because FaceWall won't appear in the library view. Only those properties from the base class Element can be used.